test/functional/mempool_packages.py

```
@@ -34,8 +37,8 @@ def set_test_params(self):
             ],
             [
                 "-maxorphantxsize=1000",
-                "-limitancestorcount={}".format(MAX_ANCESTORS_CUSTOM),
-                "-limitdescendantcount={}".format(MAX_DESCENDANTS_CUSTOM),
+                "-limitancestorcount={}".format(CUSTOM_ANCESTOR_LIMIT),
```

Ruff (0.12.2)

Use f-string instead of `format` call

Convert to f-string

(UP032)